supabase/migrations/44_draft_associated_collections.sql

```diff
@@ -0,0 +1,74 @@
+begin;
+
+-- Update permissions for live_spec_flows to allow users to directly access it as long as they
+-- have access to at least one side. This may allow users to know the id of specifications that
+-- they are not authorized to, but nothing else.
+alter policy "Users must be authorized to referenced specifications" on live_spec_flows
+  rename to "Users must be authorized to one referenced specification";
+
+alter policy "Users must be authorized to one referenced specification" on live_spec_flows
```

Comment:
This policy differs from what I thought we landed at:

* I'm able to see an edge if I have **admin** to one side.
* I'm able to see an edge if I have read of **both** sides.
* I should **not** see all other tasks B, C, D that use spec A just because I, too, have read to A.

It's not the biggest deal because you can't actually get anything but the ID's.

But it **would** be a bigger deal if this auth logic were copy/pasted into a future `live_spec_flows_ext` view.

Please at least comment how this is wrong and what it should be instead.

Reply:
:+1: commented